content/doc/tutorials/tutorial-for-installing-jenkins-on-AWS.adoc

```diff
@@ -376,9 +360,30 @@ A collection of new fields appears.
 +
 image::tutorials/AWS/add-amazon-cloud.png[Adding Amazon EC2 cloud to Jenkins.]
 
-. Fill out all the fields.
-You will need to input the *Amazon EC2 Credentials*, ensuring they are *AWS Credentials*.
-
+. Click Add under Amazon EC2 Credentials
++
+image::tutorials/AWS/configure_cloud_add_ec2_credentials.png[Adding EC2 credentials in Configure Cloud.]
+.. From the Jenkins Credentials Provider, choose AWS Credentials as the Kind
++
+image::tutorials/AWS/jenkins_credentials_provider_aws_credentials.png[Choosing Kind AWS Credentials.]
+.. Scroll down and enter in the IAM User programmatic access keys with permissions to launch EC2 instances and click Add
++
+image::tutorials/AWS/add_access_secret_access_keys.png[Adding AWS Credentials.]
+.. Scroll down to select your region using the drop-down, and click Add for the EC2 Key Pair's Private Key
++
+image::tutorials/AWS/configure_cloud_region_private_key.png[Set Region and add Private Key.]
+.. From the Jenkins Credentials Provider, choose SSH Username with private key as the Kind and set the Username to ec2-user
++
+image::tutorials/AWS/ssh_username.png[Set Kind to SSH Username with private key.]
+.. Scroll down and choose Enter Directly under Private Key and click Add
++
+image::tutorials/AWS/private_key_enter_directly.png[Set Private Key to Enter Directly.]
+.. Open the private key pair you created in the <<Creating a key pair, creating a key pair>> directly and paste in the contents from "-----BEGIN RSA PRIVATE KEY-----" to "-----END RSA PRIVATE KEY-----". Click Add when completed
```

Comment:
```suggestion
.. Open the private key pair you created in the <<Creating a key pair, creating a key pair>> step and paste in the contents from "-----BEGIN RSA PRIVATE KEY-----" to "-----END RSA PRIVATE KEY-----".
Select Add when completed.
```
small syntax and formatting suggestion, just to make sure the instructions are deliberate in what they say